src/fireedge/src/modules/containers/VmGroups/VmGroups.js

```diff
@@ -0,0 +1,223 @@
+export function VmGroups() {
+  const [selectedRows, setSelectedRows] = useState(() => [])
+  const actions = VmGroupsTable.Actions({ selectedRows, setSelectedRows })
+  const { zone } = useGeneral()
+
+  return (
+    <TranslateProvider>
+      <ResourcesBackButton
+        selectedRows={selectedRows}
+        setSelectedRows={setSelectedRows}
+        useUpdateMutation={VmGroupAPI.useUpdateVMGroupMutation}
+        zone={zone}
+        actions={actions}
+        table={(props) => (
+          <VmGroupsTable.Table
+            onSelectedRowsChange={props.setSelectedRows}
+            globalActions={props.actions}
+            useUpdateMutation={props.useUpdateMutation}
+            onRowClick={props.resourcesBackButtonClick}
+            zoneId={props.zone}
+            initialState={{
+              selectedRowIds: props.selectedRowsTable,
+            }}
+          />
+        )}
+        simpleGroupsTags={(props) => (
+          <GroupedTags
+            tags={props.selectedRows}
+            handleElement={props.handleElement}
+            onDelete={props.handleUnselectRow}
+          />
+        )}
+        info={(props) => {
+          const propsInfo = {
+            user: props?.selectedRows?.[0]?.original,
+            selectedRows: props?.selectedRows,
+          }
+          props?.gotoPage && (propsInfo.gotoPage = props.gotoPage)
+          props?.unselect && (propsInfo.unselect = props.unselect)
+
+          return <InfoTabs {...propsInfo} />
+        }}
+      />
+    </TranslateProvider>
+  )
+}
+
+/**
+ * Displays details of an VmGroup.
+ *
+ * @param {VmGroup} user - VmGroup to display
+ * @param {Function} [gotoPage] - Function to navigate to a page of an VmGroup
+ * @param {Function} [unselect] - Function to unselect a VmGroup
+ * @param {object[]} [selectedRows] - Selected rows (for Labels)
+ * @returns {ReactElement} VmGroup details
+ */
+const InfoTabs = memo(({ user, gotoPage, unselect, selectedRows }) => {
+  const [get, { data: lazyData, isFetching }] =
+    VmGroupAPI.useLazyGetVMGroupQuery()
+  const id = user?.ID ?? lazyData?.ID
+
+  const { settings: { FIREEDGE: fireedge = {} } = {} } = useAuth()
+  const { FULL_SCREEN_INFO } = fireedge
+  const { fullViewMode } = SERVER_CONFIG
+  const fullModeDefault =
+    FULL_SCREEN_INFO !== undefined ? FULL_SCREEN_INFO === 'true' : fullViewMode
+  const { isFullMode } = useGeneral()
+  const { setFullMode } = useGeneralApi()
+
+  useEffect(() => {
+    !isFullMode && gotoPage()
+  }, [])
+
+  return (
+    <Stack overflow="auto">
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        gap={1}
+        mx={1}
+        mb={1}
+      >
+        <Stack direction="row">
+          {fullModeDefault && (
+            <SubmitButton
+              data-cy="detail-back"
+              icon={<NavArrowLeft />}
+              tooltip={Tr(T.Back)}
+              isSubmitting={isFetching}
+              onClick={() => unselect()}
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using Function.prototype.bind and arrows functions as attributes will negatively impact performance in React. Each time the parent is rendered, the function will be re-created and trigger a render of the component causing excessive renders and more memory use.